src/main/java/no/nav/fo/veilarbarena/service/OppfolgingsbrukerController.java

```diff
@@ -10,30 +12,63 @@
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
+import javax.ws.rs.WebApplicationException;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Timestamp;
 import java.time.ZoneId;
 import java.time.ZonedDateTime;
+import java.util.List;
 import java.util.Optional;
 
 @Component
-@Path("/oppfolgingsbruker/{fnr}")
+@Path("/oppfolgingsbruker")
 public class OppfolgingsbrukerController {
 
-    @Inject
-    private JdbcTemplate db;
+    static final int PAGE_SIZE_MAX = 1000;
+    private static final int PAGE_NUMBER_MAX = 500_000;
+
+    private final JdbcTemplate db;
+    private final AuthService authService;
 
     @Inject
-    private AuthService authService;
+    public OppfolgingsbrukerController(JdbcTemplate db, AuthService authService) {
+        this.db = db;
+        this.authService = authService;
+    }
+
 
     @GET
+    @Path("/{fnr}")
     public UserDTO getOppfolgingsbruker(@PathParam("fnr") String fnr){
         authService.sjekkTilgang(fnr);
-        return hentOppfolgingsbruker(fnr);
+        return hentOppfolgingsbrukere(fnr);
+    }
+
+    @GET
+    public UserPageDTO getOppfolgingsbruker(@PathParam("page_number") String pageNumberParam, @PathParam("page_size") String pageSizeParam) {
```

Comment:
Har ikke jaxrs fornuftig validering for `int`/`Integer` direkte?

Reply:
Jo det har det nok, endret på det